wakuv2/waku.go

```diff
@@ -290,7 +290,7 @@ func New(nodeKey *ecdsa.PrivateKey, fleet string, cfg *Config, logger *zap.Logge
 		node.WithTopicHealthStatusChannel(waku.topicHealthStatusChan),
 		node.WithKeepAlive(randomPeersKeepAliveInterval, allPeersKeepAliveInterval),
 		node.WithLogger(logger),
-		node.WithLogLevel(logger.Level()),
+		node.WithLogLevel(zap.WarnLevel), // Set to WarnLevel to avoid spamming app logs
```

Comment:
@richard-ramos @chaitanyaprem Can you guys please confirm this is fine with you?

Reply:
are info logs spamming from waku and go-libp2p? 
It would be good to have info level enabled by default.

Maybe it would be good to fix the info logs that are spamming. wdyt @richard-ramos ?

Reply:
I took a closer look, and indeed, the info-level logs from both Waku and libp2p account for no more than 2% of the size of geth.log. Changed the log level to INFO.